Backend/app/spotify_electron/user/base_user_service.py

```diff
@@ -50,7 +50,7 @@
 
 
 # TODO not hardcoded
-MAX_NUMBER_PLAYBACK_HISTORY_SONGS = 5
+MAX_NUMBER_PLAYBACK_HISTORY_SONGS = 50
```

Comment:
Still named playback, search for playback naming in the project and refactor it according to the new stream convention

Reply:
There should be no more occurences of word "playback" as Vscode couldnt find any other after latest commit. hopefully hehe